components/Premium/PremiumFeatures/PremiumFeatures.tsx

```diff
@@ -92,6 +92,49 @@ function PremiumFeatures() {
                         {checkIconElement}
                         {checkIconElement}
                     </tr>
+                    <tr>
```

Comment:
There should be one line with `Price estimations in game`  this image:
![image](https://github.com/Coflnet/hypixel-react/assets/16632490/5f6387b2-c3c8-4780-895e-41647688b33e)

and checks on every tier